src/payments/services/coupon.service.ts

```
@@ -409,13 +409,17 @@ export class CouponService {
-  async listCoupons(filters?: {
-    contextType?: PaymentContextType;
-    contextId?: string;
-    isActive?: boolean;
-  }): Promise<DiscountCoupon[]> {
+  async listCoupons(
+    filters?: {
+      contextType?: PaymentContextType;
+      contextId?: string;
+      isActive?: boolean;
+    },
+    limit: number = 50,
```

ESLint

[error] Type number trivially inferred from a number literal, remove type annotation.

(`@typescript-eslint/no-inferrable-types`)

---

[error] Type number trivially inferred from a number literal, remove type annotation.

(`@typescript-eslint/no-inferrable-types`)

---

[error] Delete `,`

(prettier/prettier)